### ISSUE TITLE
build 1.12.6 error

### ISSUE THREAD

**Issue by naiba** [techknowlogick/xgo#24]
### base Dockerfile
https://github.com/gophers/xgo/tree/master/docker/base
### 1.12.6 Dockerfile
https://github.com/gophers/xgo/tree/master/docker/go-1.12.6
### Build log
```
Building in Docker Cloud's infrastructure...
Cloning into '.'...
Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
Reset branch 'master'
Your branch is up-to-date with 'origin/master'.
KernelVersion: 4.4.0-1060-aws
Components: [{u'Version': u'18.03.1-ee-3', u'Name': u'Engine', u'Details': {u'KernelVersion': u'4.4.0-1060-aws', u'Os': u'linux', u'BuildTime': u'2018-08-30T18:42:30.000000000+00:00', u'ApiVersion': u'1.37', u'MinAPIVersion': u'1.12', u'GitCommit': u'b9a5c95', u'Arch': u'amd64', u'Experimental': u'false', u'GoVersion': u'go1.10.2'}}]
Arch: amd64
BuildTime: 2018-08-30T18:42:30.000000000+00:00
ApiVersion: 1.37
Platform: {u'Name': u''}
Version: 18.03.1-ee-3
MinAPIVersion: 1.12
GitCommit: b9a5c95
Os: linux
GoVersion: go1.10.2
Starting build of index.docker.io/naiba/xgo:go-1.12.6...
Step 1/3 : FROM naiba/xgo:base
---> 0783b60d1b67
Step 2/3 : ENV GO_VERSION 1126
---> Running in dcd9e3df2915
Removing intermediate container dcd9e3df2915
---> 038734035d51
Step 3/3 : RUN export ROOT_DIST=https://storage.googleapis.com/golang/go1.12.6.linux-amd64.tar.gz && export ROOT_DIST_SHA=dbcf71a3c1ea53b8d54ef1b48c85a39a6c9a935d01fc8291ff2b92028e59913c && $BOOTSTRAP_PURE
---> Running in 027a6c6d44c9
Downloading https://storage.googleapis.com/golang/go1.12.6.linux-amd64.tar.gz...
go1.12.6.linux-amd64.tar.gz: OK
Bootstrapping linux/386...
Bootstrapping linux/arm64...
Bootstrapping linux/mips64...
Bootstrapping linux/mips64le...
Bootstrapping linux/mips...
Bootstrapping linux/mipsle...
Bootstrapping windows/amd64...
Bootstrapping windows/386...
Bootstrapping darwin/amd64...
# runtime/cgo
/usr/local/osx-ndk-x86/bin/x86_64-apple-darwin15-ld: error while loading shared libraries: libxar.so.1: cannot open shared object file: No such file or directory
clang: error: unable to execute command: No such file or directory
clang: error: linker command failed due to signal (use -v to see invocation)
Removing intermediate container 027a6c6d44c9
The command '/bin/sh -c export ROOT_DIST=https://storage.googleapis.com/golang/go1.12.6.linux-amd64.tar.gz && export ROOT_DIST_SHA=dbcf71a3c1ea53b8d54ef1b48c85a39a6c9a935d01fc8291ff2b92028e59913c && $BOOTSTRAP_PURE' returned a non-zero code: 2
```

**Comment by kolaente** [comment]
Can you give more informations?

**Comment by naiba** [comment]
> Can you give more information?

Hello @kolaente, I've updated infomation detail.

**Comment by kolaente** [comment]
Does it work when you use the `techknowlogick/xgo` docker images?

**Comment by naiba** [comment]
@kolaente I merged your PR and built the image.

**Comment by kolaente** [comment]
Which pr?

**Comment by naiba** [comment]
> Which pr?

@kolaente #17

**Comment by kolaente** [comment]
As I said in that pr, it is not finished yet. I would prefer to finish that one first and then check again.

**Comment by naiba** [comment]
@kolaente Can't wait for this, can you fix docker build error first?

**Comment by kolaente** [comment]
@naiba I can try to debug this, but it seems some dependency upstream changed which means we can't build the darwin compiler anymore. Unfortunatly, I don't have much time currently, but I see what I can do. If you have any input, PR's welcome :)

**Comment by kolaente** [comment]
I'll be tracking the fix in #25

**Comment by naiba** [comment]
@kolaente Thank you very match, I alse try to fix this by hack osxcross.
